tests/test_arg_parsing.py

```diff
@@ -24,19 +25,43 @@ def test_peal_speed_parsing_errors(self):
                       ("-4h", "The hour value '-4' must be a positive integer."),
                       ("3hP", "The minute value 'P' is not an integer."),
                       ("3h-2m", "The minute value '-2' must be a positive integer."),
-                      ("3h100m", "The minute value '100' must be smaller than 60."),
+                      ("3h100", "The minute value '100' must be smaller than 60."),
                       ("125x", "The minute value '125x' is not an integer."),
                       ("-2m", "The minute value '-2' must be a positive integer."),
                       ("-200", "The minute value '-200' must be a positive integer."),
                       ("", "The minute value '' is not an integer."),
                       ("    ", "The minute value '' is not an integer."),
                       ("\nXX   X ", "The minute value 'XX   X' is not an integer.")]
+
         for (input_arg, expected_message) in test_cases:
             with self.subTest(input=input_arg, expected_message=expected_message):
                 with self.assertRaises(PealSpeedParseError) as e:
                     parse_peal_speed(input_arg)
                 self.assertEqual(expected_message, e.exception.message)
 
+    def test_call_parsing(self):
+        test_cases = [("14", {0: '14'}),
+                      ("  0 \t:  \n 16   ", {0: '16'}),
+                      ("20: 70", {20: '70'}),
+                      ("20: 70, 14", {20: '70', 0: '14'})]
```

Comment:
> I couldn't have told you what `20: 70, 14` did on first reading it.
> 
>     1. Possibly enforce a location on all sections once more than one?

Maybe - though I like how concise this is.  Maybe I'll just update all the examples so that they always have the call locations specified - that way if people want to cut corners then they can, since they should know what they want to say.

>     2. `convert_pn` uses `,` to separate blocks of placenotation so `4: 3.1,5` actually reads like `{4: '3.1.5'}` not `{4: '3.1', 0:'5'}` and `4: &3.1,5` would be `{4: '3.1.3.5'}`. Would `;` work as a seperator?

This is a very good point - `,` is probably a bad choice for a delimiter.  I think `;` is also too similar to `:`, so how about using `/` or some other char that has no meaning in place notation parsing?  We could potentially discuss this with all the other things...

Reply:
Updated to using `/`